Source/Events/BrowseResultsAdded.lua

```diff
@@ -23,6 +23,6 @@ function Auctionator.Events.BrowseResultsAdded(addedBrowseResults)
 
   -- We don't use the C_AuctionHouse.HasFullBrowseResults as it doesn't work.
   if (#addedBrowseResults > 0) then
-    C_AuctionHouse.RequestMoreBrowseResults()
+--    C_AuctionHouse.RequestMoreBrowseResults()
```

Comment:
Why is this commented out?

Reply:
Oh, I was debugging replicate and that scan method dragged in so many items that I couldn't see when replicate added an item wrong.

Will fix.